test/testcafe/spec/SelectAuthenticatorForEnroll_spec.js

```diff
@@ -236,10 +253,12 @@ test.requestHooks(requestLogger, mockEnrollAuthenticatorPassword)('select passwo
   await t.expect(req2.method).eql('post');
   await t.expect(req2.body).eql('{"authenticator":{"id":"autwa6eD9o02iBbtv0g3"},"stateHandle":"02CqFbzJ_zMGCqXut-1CNXfafiTkh9wGlbFqi9Xupt"}');
 
-  const req3 = requestLogger.requests[2].request;
-  await t.expect(req3.url).eql('http://localhost:3000/idp/idx/credential/enroll');
-  await t.expect(req3.method).eql('post');
-  await t.expect(req3.body).eql('{"authenticator":{"id":"autwa6eD9o02iBbtv0g3"},"stateHandle":"02CqFbzJ_zMGCqXut-1CNXfafiTkh9wGlbFqi9Xupt"}');
+  if (!process.env.OKTA_SIW_V3) {
```

Comment:
why v3 not making this request? or just the request order is changed?

Reply:
please also add a comment to explain why this spec should not happen in v3

Reply:
I will add a comment, also adding to the parity test wiki. 

But the reason is because v2 only makes 3 requests none of which are for the "back to authenticator list" click (v2 does not make a request for this click). v3 makes 4 requests, where the "back to authenticator list" is included, req3 for v3 is the back to authenticator list which only includes the stateHandle in the payload so this would fail on v3.

Reply:
ic, switch authenticator is a request in the middle, so it's about the ordering